src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -249,9 +249,11 @@
         (throw e)))))
 
 (defn- prepared-statement*
-  ^PreparedStatement [driver conn sql params canceled-chan]
+  ^PreparedStatement [driver fetch-size conn sql params canceled-chan]
```

Comment:
this might be a place to use optional args, like:

`[driver conn sql params canceled-chan & {:keys [fetch-size]}]`

and then when you call it:

`(prepared-statement* driver conn sql params (context/canceled-chan context) :fetch-size fs)`

Reply:
That is better (fixed).

Reply:
(actually, irrelevant now with latest changes)